src/debug-adapter/debugEventManager.ts

```diff
@@ -135,10 +139,11 @@ export class DebugEventManager {
     variable: DebugProtocol.Variable
   ): Promise<[variable: PanelViewVariable, isNewAndObject: boolean]> {
     let isNewAndObject = false;
-    const panelViewVariable: PanelViewVariable = { id: id !== 'null' ? id : hash(variable) };
+    const panelViewVariable: PanelViewVariable = { id: id !== 'null' ? id : `${DebugEventManager.variablePrefix}${hash(variable)}` };
```

Comment:
Is the prefix (none or `DebugEventManager.objectPrefix`) always consistent with `isNewAndObject` (2/2)?